### PR TITLE
[engraving] Added layout stable version. Step 30

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/engraving/CMakeLists.txt [hunk 1]
```diff
@@ -23,6 +23,8 @@ set(MODULE_QRC engraving.qrc)
 
 include(${CMAKE_CURRENT_LIST_DIR}/libmscore/libmscore.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/compat/midi/compatmidi.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/layout/v0/layout.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/layout/v20230717/layout.cmake)
```
Comment (RomanPudashkin) [inline src/engraving/CMakeLists.txt:27]: Maybe we should just call them stable/unstable? seems to me that these numbers will confuse developers a lot. And also, we probably only need two versions of the layout: main and under development

Comment (igorkorsukov) [reply src/engraving/CMakeLists.txt]: Renamed

File: src/engraving/engravingmodule.cpp [hunk 2]
```diff
@@ -76,7 +77,8 @@ void EngravingModule::registerExports()
 #endif
 
     // internal
-    ioc()->registerExport<layout::ILayout>(moduleName(), new layout::v0::Layout());
+    ioc()->registerExport<layout::ILayout>(moduleName(), new layout::dev::Layout());
+    //ioc()->registerExport<layout::ILayout>(moduleName(), new layout::v20230717::Layout());
```
Comment (RomanPudashkin) [inline src/engraving/engravingmodule.cpp:81]: Could you please update this comment?

File: src/palette/internal/palettelayout.cpp [hunk 3]
```diff
@@ -359,8 +359,8 @@ void PaletteLayout::layout(Ambitus* item, const Context& ctx)
 
 void PaletteLayout::layout(Arpeggio* item, const Context& ctx)
 {
-    engraving::layout::v0::LayoutContext ctxv0(ctx.dontUseScore());
-    engraving::layout::v0::ArpeggioLayout::layout(item, ctxv0);
+    engraving::layout::stable::LayoutContext ctxv0(ctx.dontUseScore());
```
Comment (RomanPudashkin) [inline src/palette/internal/palettelayout.cpp:362]: I think it should be just ctx (instead of ctxv0). Here and in other places too

**Review by RomanPudashkin — APPROVED** [review]

File: src/palette/internal/palettelayout.cpp (hunk 3 above)
Comment (igorkorsukov) [reply src/palette/internal/palettelayout.cpp:362]: There is already a variable called `ctx`

**Pull request merged**